### PR TITLE
#23: test_12 config cypress

### PR DESCRIPTION
**Pull request by TheMindset**
Test for config cypress with circleci
Closes: #23

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -29,6 +29,7 @@ gem 'uglifier', '>= 1.3.0'
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'factory_bot_rails', '~> 4.0'
+  gem 'shoulda-matchers', '~> 3.1'
```
Comment (ocardinaux) [inline Gemfile]: why moved here, not in test group ?

File: coverage/.resultset.json [hunk 2]
```diff
@@ -0,0 +1,254 @@
+{
```
Comment (ocardinaux) [inline coverage/.resultset.json]: files in coverage are in .gitignore

File: cypress/integration/examples/actions.spec.js [hunk 3]
```diff
@@ -0,0 +1,272 @@
+/// <reference types="Cypress" />
```
Comment (ocardinaux) [inline cypress/integration/examples/actions.spec.js]: should these example files go in .gitignore too?

File: Gemfile (hunk 1 above)
Comment (TheMindset) [reply Gemfile]: because in dev environment, rspec don't work without shoulda_matchers

Comment (TheMindset) [reply Gemfile]: in moved here for have shoulda_matchers in all environment test and dev

File: coverage/.resultset.json (hunk 2 above)
Comment (TheMindset) [reply coverage/.resultset.json]: yes

File: cypress/integration/examples/actions.spec.js (hunk 3 above)
Comment (TheMindset) [reply cypress/integration/examples/actions.spec.js]: its done

**Pull request merged**